app/timetables/tests/test_models.py

```diff
@@ -148,21 +148,47 @@ def test_duplicate_api_key_cannot_be_saved(self):
             api_key='419223',
             cycle_length=14,
             current_cycle_day=2,
-            date_created=datetime.time(datetime.strptime('4:7:9', '%H:%M:%S')),
-            date_modified=datetime.time(datetime.strptime('15:7:9', '%H:%M:%S'))
+            date_created=datetime.strptime('05 07 2016', '%d %m %Y'),
+            date_modified=datetime.strptime('06 08 2016', '%d %m %Y')
         )
 
-        self.assertRaises(IntegrityError, timetable.save)
+        self.assertRaises(ValidationError, timetable.save)
 
-    def test_timetable_with_current_cycle_day_greater_than_cycle_length_cannot_be_saved(self):
+    def test_current_cycle_day_greater_than_cycle_length_cannot_be_saved(self):
         timetable = Timetable(
             name='timtable',
             code='FBI232123',
             api_key='4192237',
             cycle_length=1,
             current_cycle_day=2,
-            date_created=datetime.time(datetime.strptime('4:7:9', '%H:%M:%S')),
-            date_modified=datetime.time(datetime.strptime('15:7:9', '%H:%M:%S'))
+            date_created=datetime.strptime('05 07 2016', '%d %m %Y'),
```

Comment:
Initially, we had a datetime.time object, which is not acceptable for a datetime.date field. When I called the full_clean method, the test threw errors because it was expecting a datetime object and not just a time object.